src/routes/+page.svelte

```diff
@@ -25,17 +26,85 @@
 								dynamic storytelling.
 							</h2>
 						</div>
-						<div class="mt-16 flex justify-end">
-							<Button class="bg-tertiary-400 w-1/2 rounded-none py-4 font-primary !text-4xl !text-quaternary lg:w-1/4" href="/try-for-free"
-								>TRY FOR FREE</Button
+						<div class="mt-16 flex justify-end ">
+							<Button class="bg-tertiary-400 w-1/2 rounded py-4 font-primary !text-4xl !text-quaternary lg:w-1/4" href="/try-for-free"
+								>PLAY NOW</Button
 							>
-						</div>
-					</div>
+						</div>			
+		</section>
+		<div class=" inset-0 flex flex-col items-center justify-start gap-12 p-9 ">
+			<div class="w-full max-w-4xl p-16 bg-tertiary-400 font-primary rounded shadow-lg">
+			  <h2 class="text-4xl font-extrabold text-tertiary uppercase">
```

Comment:
You can increase the letter spacing so the letters are not bunched together as much: https://tailwindcss.com/docs/letter-spacing
tracking-wide is probably good here.